students/yesol/westagram/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class Member(models.Model):
+    name     = models.CharField(max_length=20)
+    email    = models.CharField(max_length=50)
+    password = models.CharField(max_length=20)
+    phone    = models.CharField(max_length=30)
```

Comment:
```suggestion
    phone_number    = models.CharField(max_length=30)
```
변수명은 확실하게 나타내주는 것이 좋습니다.